test/remote/usps_test.rb

```diff
@@ -225,6 +225,19 @@ def test_must_provide_login_creds_when_instantiating
     end
   end
 
+  def test_delivery_range_when_providing_ship_date
+    response = @carrier.find_rates(
+       location_fixtures[:beverly_hills],
+       location_fixtures[:new_york],
+      Package.new(16, [12, 6, 2], :units => :imperial),
```

Comment:
I know that colon syntax is a common critique. It felt weird to mix it in this test file.
So maybe that's my next pull request to clean up this test file.